packages/carbon-graphs/src/js/controls/Graph/helpers/helpers.js

```diff
@@ -167,10 +167,17 @@ const translateVerticalGrid = (axis, config) => {
     : constants.TICK_ORIENTATION.BOTTOM;
   if (utils.notEmpty(config.axis.x.ticks.values)) {
     const ticks = config.axis.x.ticks.values;
-    xAxisGrid = axis.x
-      .tickValues(processTickValues(ticks))
-      .tickSize(getYAxisHeight(config) * tickSizeMultiplicand, 0, 0)
-      .tickFormat('');
+    if (config.axis.x.domain === undefined) {
```

Comment:
We can use the `utils.isUndefined` method 

https://github.com/cerner/terra-graphs/blob/5688742718cc2e135e549e233d729f6c739ee1f9/packages/carbon-graphs/src/js/helpers/utils.js#L48-L55

```suggestion
    if (utils.isUndefined(config.axis.x.domain)) {
```